### ISSUE TITLE
Person form add zip code look up

### ISSUE THREAD

**Issue by VanAuken** [ReadyResponder/ReadyResponder#98]
On '**New Person Form**', and edit '**Person**' form, please add feature that '**city**' updates according to '**zip code**' entry.

This will make data entry easier for user, as they won't need to type the city.  It will also produce an opportunity for the user to recognize if they have misread the applicants handwriting.

**Comment by kgf** [comment]
Should this also update the state field ?

**Comment by kgf** [comment]
Ideally this should reference an online data source via AJAX

**Comment by kgf** [comment]
USPS does seem to have an API for this
https://www.usps.com/business/web-tools-apis/address-information-api.htm

**Comment by OrthoDex** [comment]
Hey! I'd like to take this up if it's alright, we can use the Google Geocode API: https://developers.google.com/maps/documentation/geocoding/start?csw=1

**Comment by kgf** [comment]
@OrthoDex That would be great ! Thanks. 

Please reference the API key using env vars, similar to this:

```
# create Twilio Client
    account_sid = ENV["TWILIO_ACCOUNT_SID"] # Your Account SID from www.twilio.com/console
    auth_token = ENV["TWILIO_AUTH_TOKEN"]   # Your Auth Token from www.twilio.com/console
```

Please also scope them to google, like GOOGLE_ACCOUNT_API_KEY
I hope you'll be able to get this to work when giving Google only the zipcode.

Thanks again for your interest.

**Comment by OrthoDex** [comment]
Alright! I'll issue a pull request soon. :smile:

**Comment by kgf** [comment]
I'm looking forward to it, and so is @VanAuken !

**Comment by kgf** [comment]
I am mentioning this on all the claimed issues. We always appreciate people's help, but if you could have any PRs submitted by 1PM EST this Sunday, I'll have time to review and integrate them for a scheduled demo on Monday.

Thanks !
Kevin

**Comment by kgf** [comment]
Fixed by #218
